File: Level 1/가운데 글자 가져오기/Solution.java
class Solution {
    public String solution(String s) {
        int length = s.length();
        int middle = length/2;
        String answer = "";
        if(length%2==0) {
            answer = s.substring(middle-1, middle+1);
        }
        else answer = s.substring(middle, middle+1);
        return answer;
    }
}

//출처: 프로그래머스 코딩 테스트 연습, https://programmers.co.kr/learn/challenges
//Hajin Kim, 2021-05-03

File: Level 1/가운데 글자 가져오기/solution.js
function solution(s) {
    var answer = '';
    let start = Math.floor(s.length/2);
    if(s.length%2 == 0) {
        answer = s.slice(start-1, start+1);
    }
    else {
        answer = s.slice(start, start+1);
    }
    return answer;
}